e2e/steps_file_LRspec.js

```diff
@@ -332,8 +330,6 @@ module.exports = function () {
                  () => specInterestDateStartPage.selectInterestDateStart(),
                  () => specInterestDateEndPage.selectInterestDateEnd(),
                  () => this.clickContinue(),
-                 () => pbaNumberPage.selectPbaNumber(),
```

Comment:
I wonder if we should put the changes in tests under a featureToggle.
There is a variable you could use to check if pbav3 is on: const {checkToggleEnabled} = require('./testingSupport');

Reply:
Done